code/game/objects/items/devices/instruments.dm

```diff
@@ -164,3 +198,60 @@
 	throw_range = 15
 	hitsound = 'sound/items/bikehorn.ogg'
 
+//spooky stuff
+/obj/item/device/instrument/brass/trumpet/spectral/attack(mob/living/carbon/C, mob/user)
+	playsound (loc, 'sound/instruments/trombone/En4.mid', 100,1,-1)
+	spectral_attack(C, user)
+	..()
+
+/obj/item/device/instrument/brass/saxophone/spectral/attack(mob/living/carbon/C, mob/user)
+	playsound (loc, 'sound/instruments/saxophone/En4.mid', 100,1,-1)
+	spectral_attack(C, user)
+	..()
+
+/obj/item/device/instrument/brass/trombone/spectral/attack(mob/living/carbon/C, mob/user)
+	playsound (loc, 'sound/instruments/trombone/Cn4.mid', 100,1,-1)
+	spectral_attack(C, user)
+	..()
+
+//spooky procs
+/obj/item/device/instrument/brass/proc/spectral_attack(mob/living/carbon/C, mob/user)
+	if(ishuman(user)) //this weapon wasn't meant for mortals.
+		var/mob/living/carbon/human/U = user
+		if(!istype(U.dna.species, /datum/species/skeleton))
+			U.adjustStaminaLoss(25) //Extra Damage
+			attack(user)
+			U.Knockdown(30)
+			to_chat(U, "<span class= 'danger'> Your ears weren't meant for this spectral sound.</span>")
+			return ..()
+
+	if(ishuman(C))
+		var/mob/living/carbon/human/H = C
+		if(istype(H.dna.species, /datum/species/skeleton))
+			return ..() //undead are unaffected by the spook-pocalypse.
+		if(istype(H.dna.species, /datum/species/zombie))
+			H.adjustStaminaLoss(25) //zombies can't resist the doot
+		C.Knockdown(15)
+		C.Jitter(15)
+		C.stuttering = 20
+		C.adjustStaminaLoss(25) //only humanoids lose the will to live
+		to_chat(C, "<font color='red' size='4'><B>DOOT</B></span>")
+
+		if((H.getStaminaLoss() > 95) && (!istype(H.dna.species, /datum/species/skeleton)) && (!istype(H.dna.species, /datum/species/golem)) && (!istype(H.dna.species, /datum/species/android)) && (!istype(H.dna.species, /datum/species/jelly)))
+			H.Knockdown(30) //doot!
+			H.set_species(/datum/species/skeleton)
+			H.visible_message("<span class='warning'>[H] has given up on life as a mortal.</span>")
+			to_chat(H, "<B>You are the spooky skeleton!</B>")
+			to_chat(H, "A new life and identity has begun. Help your fellow skeletons into bringing out the spooky-pocalypse. You haven't forgotten your past life, but you are no longer beholden to any past loyalties.")

```

Comment:
I believe you need admin ruling to be able to say "Past allegiances are foregone" in this text unless there's something I don't know going about.

Reply:
They can only be spawned by admins anyway.